packages/react-native-bpk-component-select/src/BpkSelect.js

```diff
@@ -159,7 +166,7 @@ const BpkSelect = (props: Props) => {
         {showImage && styledImage}
         {content || label}
         {validityIcon}
-        <SelectIcon />
+        <SelectIcon disabled={disabled} />
```

Comment:
How come this has changed?

Reply:
This was wrong, the icon had no disabled state before. I fixed it as part of this PR

Reply:
Cool - nice one 😉